airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -915,7 +915,7 @@ def _bq_cast(string_field, bq_type):
     if string_field is None:
         return None
     elif bq_type == 'INTEGER' or bq_type == 'TIMESTAMP':
-        return int(string_field)
+        return int(float(string_field))
```

Comment:
Why need to convert to float -> int ?

Reply:
It does not. We do not do this conversion in the google-cloud-bigquery library.

https://github.com/GoogleCloudPlatform/google-cloud-python/blob/75de6a53c9feee1bc37ee4bfd1369b3010403194/bigquery/google/cloud/bigquery/_helpers.py#L38-L41

TIMESTAMP can contain fractional values, so perhaps that's why this was added? Probably best to separate these types.

Reply:
Thanks @tswast for picking this up after the long silence. @dlackty Could you create a PR for this? By splitting out the `TIMESTAMP` as suggested.

Reply:
@Fokko & @tswast for some reason I didn't see your comments. I'll open another PR for this.

Reply:
Thanks @dlackty